ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -157,47 +157,54 @@ const BaseSearchLinks = React.memo(({ variant, mainTranscript, genesById }) => {
+      links.push({ name: 'google', href: `https://www.google.com/search?q=(${geneNames.join('|')})+(${variations.join('|')}` })
     }
 
-    addDividedLink(links, 'pubmed', `https://www.ncbi.nlm.nih.gov/pubmed?term=${pubmedSearch}`)
+    links.push({ name: 'pubmed', href: `https://www.ncbi.nlm.nih.gov/pubmed?term=${pubmedSearch}` })
   }
 
-  const seqrLinkProps = { genomeVersion: variant.genomeVersion, svType: variant.svType }
-  if (variant.svType) {
-    if (variant.endChrom && variant.endChrom !== variant.chrom) {
-      seqrLinkProps.location = `${variant.chrom}:${variant.pos - 50}-${variant.pos + 50}`
-    } else {
-      seqrLinkProps.location = `${variant.chrom}:${variant.pos}-${variant.end}%20`
+  const isSv = !!svType
+  if (isSv) {
+    const useLiftover = liftedOverGenomeVersion === GENOME_VERSION_37
+    if (genomeVersion === GENOME_VERSION_37 || (useLiftover && liftedOverPos)) {
+      const endOffset = endChrom ? 0 : end - pos
+      const start = useLiftover ? liftedOverPos : pos
+      const region = `${chrom}-${start}-${start + endOffset}`
+      links.push({ name: 'gnomAD', href: `https://gnomad.broadinstitute.org/region/${region}?dataset=gnomad_sv_r2_1` })
     }
-
-    const useLiftover = variant.liftedOverGenomeVersion === GENOME_VERSION_37
-    if (variant.genomeVersion === GENOME_VERSION_37 || (useLiftover && variant.liftedOverPos)) {
-      const endOffset = variant.endChrom ? 0 : variant.end - variant.pos
-      const start = useLiftover ? variant.liftedOverPos : variant.pos
-      const region = `${variant.chrom}-${start}-${start + endOffset}`
-      addDividedLink(links, 'gnomAD', `https://gnomad.broadinstitute.org/region/${region}?dataset=gnomad_sv_r2_1`)
+  } else if (chrom === 'M') {
+    links.push({ name: 'mitomap', href: 'https://www.mitomap.org/foswiki/bin/view/Main/SearchAllele' })
+    if (isTrnaOrRrna(genesById)) {
+      links.push({ name: 'Mitovisualize', href: `https://www.mitovisualize.org/variant/m-${pos}-${ref}-${alt}` })
     }
   } else {
-    seqrLinkProps.variantId = variant.variantId
+    links.push(
+      {
+        name: 'Geno2MP',
+        href: `https://geno2mp.gs.washington.edu/Geno2MP/#/gene/${chrom}:${pos}/chrLoc/${pos}/${pos}/${chrom}`,
+      },
+      { name: 'Iranome', href: `http://www.iranome.ir/variant/${chrom}-${pos}-${ref}-${alt}` },
+    )
   }
-  links.unshift(
+
+  return [
     <Popup
       key="seqr-search"
-      trigger={<SearchResultsLink key="seqr" buttonText="seqr" {...seqrLinkProps} />}
-      content={`Search for this variant across all your seqr projects${variant.svType ? '. Any structural variant with ≥20% reciprocal overlap will be returned.' : ''}`}
+      trigger={(
+        <SearchResultsLink
+          buttonText="seqr"
+          genomeVersion={genomeVersion}
+          svType={svType}
+          variantId={isSv ? null : variantId}
+          location={svType && (
```

Comment:
Use `isSv` instead of `svType`?

Reply:
if its not an sv we want this to be null not false